src/Automation/Interfaces/IScanner.cs

```diff
@@ -24,7 +25,7 @@ public interface IScanner
         /// </remarks>
         /// <returns>Information about the scan and any issues detected</returns>
         /// <exception cref="AxeWindowsAutomationException"/>
-        ScanResults Scan();
+        IReadOnlyCollection<ScanResults> Scan();
```

Comment:
This is a breaking change for the public API, is it fine to just change it or is there a better way to do this and prevent breaking customers code.

Reply:
Also, I'm inclined to say that leaving this as read only collection is advantages since customers might want to verify that every window is scanned or want to iterate using for loops (though in that case, LINQ is probably also needed to achieve it with index based loops).

Reply:
Good catch on the breaking change--that's something we want to avoid. Maybe we add a new entry point, something like:
```
IReadOnlyCollection<ScanResult> Scan(string scanId, bool enableMultipleWindows)
```
then we refactor the existing Scan implementations to become wrappers for the new one (and the existing entry points pass in false for `enableMultipleWindows`. We could even add an `EnableMultipleWindows` property to the `Config` class and propagate the control of the behavior all the way to the CLI command line.

Reply:
That sounds like a good idea, I'll add the second method than and also add a new field to the config class.

Reply:
Changes are implemented now. I also reworded the documentation slightly, but of course I would love the teams input on this.